src/toggle/toggle.tsx

```diff
@@ -122,6 +185,177 @@ export const Toggle = ({
         );
     };
 
+    const renderCompositeOptionSection = () => {
+        if (!compositeOptionSection) {
+            return null;
+        }
+        const ChildrenisFinalItem = !collapsible || removable;
+        return (
+            <Children
+                $selected={showMore}
+                $isNotFinalItem={
+                    ChildrenisFinalItem ? !ChildrenisFinalItem : selected
+                }
+                $disabled={disabled}
+            >
+                {compositeOptionSection}
+            </Children>
+        );
+    };
+
+    const renderViewMoreOrLessButton = () => {
+        const errorMssgOrChildrenShown = showMore || showErrorList;
+        return (
+            <>
+                <div
+                    style={{
+                        paddingTop:
+                            !selected || errorMssgOrChildrenShown
+                                ? "0rem"
+                                : "0.6875rem",
+                    }}
+                ></div>
+                <ViewMoreOrLessButtonContainer
+                    $show={!collapsible ? false : selected}
+                    $disabled={disabled}
+                    onClick={() => handleView()}
+                    data-testid="toggle-button"
+                >
+                    <ViewMoreOrLessButtonLabel
+                        weight="semibold"
+                        $disabled={disabled}
+                        data-testid="toggle-button-label"
+                    >
+                        {showMore ? "Show less" : "Show more"}
+                    </ViewMoreOrLessButtonLabel>
+                    {showMore ? (
+                        <ChevronUpIcon height={18} width={18} />
```

Comment:
instead of setting height and width here, use css in the parent to style the svg

```
const ViewMoreOrLessButtonContainer = styled`
  svg {
   width: 1.xrem;
   height: 1.xrem;
  }
`
```